graphics/photoqt/Portfile

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+name                photoqt
+github.setup        luspi photoqt 3.4 v
+revision            0
+categories          graphics
+license             GPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+
+description         PhotoQt Image Viewer
+long_description    PhotoQt is a simple yet powerful and good looking image viewer, based on Qt/QML, \
+                    published as open-source and completely free.
+homepage            https://photoqt.org
+
+depends_lib-append  port:desktop-file-utils \
+                    port:exiv2 \
+                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    path:lib/pkgconfig/gtk+-3.0.pc:gtk3 \
+                    port:shared-mime-info
+
+if {${subport} eq "${name}"} {
+    PortGroup       qt5 1.0
+
+    conflicts       photoqt-qt4
+    platforms       {darwin > 10}
+    checksums       rmd160  15248da2baaf763e90ebebb36b1bdabeaaa59255 \
+                    sha256  16ffaa5630d7afbe516015d228112dc224c23f1f457df8cfbe6c2500686e94ba \
+                    size    4189570
+
+    depends_lib-append \
+                    port:ImageMagick \
+                    port:kde-extra-cmake-modules \
+                    port:libarchive \
+                    port:libraw \
+                    path:lib/pkgconfig/poppler.pc:poppler \
+                    port:poppler-qt5 \
+                    port:pugixml \
+                    port:vips \
+                    port:webp
+
+    # We have to use ImageMagick here: https://github.com/luspi/photoqt/issues/19
+    configure.args-append \
+                    -DCHROMECAST=OFF \
+                    -DDEVIL=OFF \
+                    -DFREEIMAGE=OFF \
+                    -DGRAPHICSMAGICK=OFF \
+                    -DIMAGEMAGICK=OFF \
```

Comment:
based on the comment above, shouldn't this be set to ON?

Reply:
Indeed. Looks like I lost it in rebasing. Will fix now.